### DISCUSSION TITLE
Hosting issue

### DISCUSSION THREAD

**Discussion by KosmosisDire** [KosmosisDire/obsidian-webpage-export#523] [Q&A]
I have a problem: I created and hosted a piece of software with vercel and associated it with my GitHub, and I now use this plugin to export my notes: Online Web Server, and then put the generated folder directly on GitHub, but my site is not updated。我site：https://www.glacierchen.asia/.我GitHub：https://github.com/Glaciercowww/nevergiveup。

_Originally posted by @Glaciercowww in https://github.com/KosmosisDire/obsidian-webpage-export/discussions/504#discussioncomment-10815183_

**Comment by KosmosisDire** [comment]
Hello @Glaciercowww
I opened a new discussion since your issue is a seperate one from where you originally posted.

Can you step me through exactly what you did to get your site up? This is almost definetly a problem with the way you have hosted your html files.

**Reply by Glaciercoool** [reply]
I deployed vercel under this link：https://vercel.com/new/glacier-chens-projects/clone?repository-url=https%3A%2F%2Fgithub.com%2Fwalinejs%2Fwaline%2Ftree%2Fmain%2Fexample.
Then I linked a newly created GitHub repository in vercel.
Finally  use this plugin to export my notes: Online Web Server, and then put the generated folder directly on GitHub.It's no use.

**Reply by Glaciercoool** [reply]
![1727847824954](https://github.com/user-attachments/assets/55d0981d-7c47-471a-a44d-34752288a181)   （This is a mistake）

**Comment by Glaciercoool** [comment]
I have been tossing this for two days and I am very frustrated and really appreciate your reply!!~

**Reply by Glaciercoool** [reply]
I don't know if I directly put the generated files on GitHub caused the error
